core/src/luigi-config/cluster-management/cluster-management.js

```diff
@@ -53,6 +54,13 @@ export function getAfterLoginLocation(clusterName, kubeconfig) {
   }`;
 }
 
+async function trackClusterChange(params) {
```

Comment:
this will be sent even if the tracking is disabled

On a second note, React is well aware of the cluster change, why not fire it from React when a microcontext changes?